### PR TITLE
(3) fix(markdown_writer): change value for none values as -

### PR DESCRIPTION
**Pull request by Tanguylo**
* **What kind of changes does this PR introduce?** 
- [ ] Bug fix
- [ ] new features
- [ ] performance 
- [ ] docs update
- [x] Improvement

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
- [ ] Yes:
- [x] No

* **Other information**:
Change displayed value in markdown table for None values and list of list

### CODE REVIEW

**Comment by Tanguylo** [comment]
This PR makes the markdown table writer writing something in list and when value is None.
For None value, the str is '-'
For list, the str is now like that :
`str_value = str(len(list)) + ' [class_name_1, class_name_2, ..., class_name_n]'[:20]`

File: dessia_common/exports.py [hunk 1]
```diff
@@ -317,6 +317,8 @@ def _value_to_str(self, value: Any) -> str:
             return self._sequence_to_str(value)
         if isinstance(value, Dict):
             return self._dict_to_str(value)
+        if value is None:
+            return '-'
```
Comment (GhislainJ) [inline dessia_common/exports.py:321]: It looks like the returned value differs from the previous functions. The later had space around the ` - ` as this one doesn't. Is it intended ?

File: dessia_common/exports.py [hunk 2]
```diff
@@ -327,7 +329,8 @@ def _object_matrix(self, object_):
         for attr, value in object_.__dict__.items():
             matrix.append([attr,
                            value.__class__.__name__,
-                           (self._value_to_str(value) if not isinstance(value, (list, dict, set)) else ' - ')])
+                           self._value_to_str(value)])
+                           # (self._value_to_str(value) if not isinstance(value, (list, dict, set)) else ' - ')])
```
Comment (GhislainJ) [inline dessia_common/exports.py:333]: Is it important to keep this commented ?

**Review by GhislainJ — COMMENTED** [review]
Some enquiries

File: dessia_common/exports.py (hunk 2 above)
Comment (Tanguylo) [reply dessia_common/exports.py]: Removed

File: dessia_common/exports.py (hunk 1 above)
Comment (Tanguylo) [reply dessia_common/exports.py]: I'm not sure it changes something for the final render of the markdown on web app but i surrounded the - with spaces, which probably wont make the difference but gives a more consistent print

**Pull request merged**